src/app/printer-status/printer-status.component.scss

```diff
@@ -67,11 +67,98 @@
         overflow: visible;
 
         & span {
-            margin-bottom: -4vh
+            margin-bottom: -4vh;
         }
 
         & img {
             margin-top: -4vh;
         }
     }
 }
+
+.quick-control {
+    position: fixed;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.85);
+    z-index: 10;
+    opacity: 0;
+    transition: opacity ease-in-out 0.5s;
+
+    &__center-icon {
+        width: 22vw;
+        display: block;
+        text-align: center;
+        margin-top: 8vh;
+        margin-bottom: 7vh;
+        margin-left: calc(50% - 11vw);
+
+        &-small {
+            display: block;
+            width: 7.5vw;
+            margin: -9vw auto 0;
+        }
+    }
+
+    &__close {
+        display: block;
+        width: 6vw;
+        position: absolute;
+        right: 2vw;
+        top: 3vh;
+    }
+
+    &__controller {
+        width: 35vw;
+        margin: 0 auto;
+        border: solid 0.6vw;
+        border-radius: 3vw;
+        margin-top: 5vh;
```

Comment:
```suggestion
```
Part of the changes you've made seem to have moved the controller box down the screen, removing the top margin gets it back to the center again.

Reply:
Ah, it was moving the image inside the table that did it.

If you keep the image inside the table, I suggest making this change. If you move the image outside the table as in my other review comment, then this change should not be made.